src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/MinimumWageCalculator.cs

```diff
@@ -7,24 +7,35 @@ public class MinimumWageCalculator : IMinimumWageCalculator
         private const decimal WeeksPerMonth = 4m;
         private const decimal WeeksPerYear = 52m;
         private const string MissingMinWageErrorMessage = "MinWage can't be null.";
+        private const string HoursPerWeekZeroErrorMessage = "HoursPerWeek must be greater than 0.";
         private const string IncorrectWageTypeErrorMessage = "WageUnit must be either 'Weekly', 'Monthly' or 'Annually'.";
 
         public decimal CalculateMinimumWage(CreateApprenticeshipRequest request)
         {
             if (!request.MinWage.HasValue)
                 throw new ArgumentOutOfRangeException(nameof(request.MinWage), request.MinWage, MissingMinWageErrorMessage);
 
+            if (!(request.HoursPerWeek > 0))
+                throw new ArgumentOutOfRangeException(nameof(request.HoursPerWeek), request.HoursPerWeek, HoursPerWeekZeroErrorMessage);
+
```

Comment:
We have a validator for HoursPerWeek which requires a minimum of 16.  

Do we need to ensure that the HoursPerWeek validator is evaluated before we get here rather than have this guard clause?

Reply:
We've had a chat with Raina about it.
She's happy to have the additional error message output for now. (until a user complains about too many error messages).